docs/components/collapse.md

```diff
@@ -113,7 +113,7 @@ KCollapse can be controlled with `v-model`.
 </KCard>
 
 ```html
-<div class="mb-3">
+<div class="collapse-margin-bottom">
```

Comment:
issue: we don't want to name classes things that are similar to utilities.

You could have a class here like `collapse-label-wrapper` etc. or whatever makes sense. Same comment for whole PR

Reply:
I still don't think we want to build in classes like that; instead, we should have reusable SCSS mixins that can be applied _within_ a class.

Reply:
So as an example for your horizontal spacing:

```scss
@mixin horizontal-spacing {
  display: flex;
  column-gap: 4px;
}

// usage
.my-component-class {
  // other rules...
  @include horizontal-spacing;
}
```

Reply:
This way, we don't end up with utilities exposing themselves into our class names

Reply:
Hmm I like that. @aanchalm01 let me know if you want to pair to tackle this :)

Reply:
I have pushed the recent changes.